src/plugins/intel_gpu/src/kernel_selector/kernels/softmax/softmax_kernel_bf.cpp

```diff
@@ -115,7 +95,9 @@ JitConstants SoftmaxKernel_bf::GetJitConstants(const softmax_params& params, Dis
         const std::string lws_0 = "get_local_size(0)";
         const std::string data_set_count = "(FLATTEN_BF?" + toVectorMulString({f, b}) + ":" + b + ")";
         const std::string data_set_size = "(FLATTEN_BF?" + y + ":" + toVectorMulString({x, y, z, f}) + ")";
-        constexpr size_t stack_size = 100;
+        // It can be expected that the maximum possible itemsNum will not exceed 32
+        // Therefore, in dynamic shape, stack_size including additional buffer is set to 33
+        constexpr size_t stack_size = 33; // The size of stack for my_chunk
```

Comment:
Is this variable being used?

Reply:
Yes, it's used as value to generate `STACK_SIZE` jit constant.